app/src/main/java/com/realityexpander/tasky/agenda_feature/data/common/convertersDTOEntityDomain/ReminderConverters.kt

```diff
@@ -17,7 +17,7 @@ fun AgendaItem.Reminder.toEntity(): ReminderEntity {
         description = description,
         remindAt = remindAt,
         time = time,
-        isDeleted = isDeleted,
+        isSynced = isSynced
```

Comment:
Don't you also need the info how it should be synced if it's not? Meaning if it needs to be created, updated or deleted?

Reply:
## FOLLOW UP QUESTION 1

I am tracking the "modificationType" in the Sync repo. I only need to keep track of the change being sent to the server.

It seems to be working well, but maybe there is something I'm missing?

Reply:
No that's fine